_posts/2021-05-21-js-waku.md

```diff
@@ -0,0 +1,171 @@
+2021-05-21-js-waku.md---
+layout: post
+name:  "Presenting JS-Waku: Waku v2 in the browser"
+title:  "Presenting JS-Waku: Waku v2 in the browser"
+date:   2021-05-21 12:00:00 +0800
+author: franck
+published: true
+permalink: /2021-05-21-js-waku
+categories: platform
+summary: "We are summarizing the Waku v2 status, introducing JS-Waku, what we achieved so far and what is next."
+discuss: 
+---
+
+After almost three months working on JS-Waku,
```

Comment:
Needs a better intro, running in browser, leading into it. No one knows what JS Waku is initially.

Something like "one of the problems of ... running in browser allows ... this lead to ..."